### PR TITLE
Watch for withdraw Stellar transaction asynchronously.

### LINKED ISSUE #93: Create an async task to watch for the Stellar transaction
We should create an async task that watches for the corresponding Stellar transaction to hit the ledger. Once it does, the transaction status needs to be changed to completed.

This issue composes #15  and aims to aid the implementation of a full withdraw flow.

### PR DESCRIPTION
**Pull request by debnil**
This PR watches asynchronously for the Stellar withdrawal transaction. It also cleans up the receipt page (for both deposit and withdrawal) and makes sure that the `withdraw` flow works end-to-end with the SEP 6 validator. It closes #93 and #94.

### CODE REVIEW

File: src/templates/transaction/more_info.html [hunk 1]
```diff
@@ -7,8 +7,10 @@
 {% block "content" %}
 <section class="section">
     <p>
-        Please deposit <strong>{{ data.amount }}</strong> of asset <strong>{{ data.asset }}</strong> to account 22093486
-        in the Bank of SEP 6.
+        <strong>{{ data.kind }}</strong> transaction with ID <strong>{{ data.id }}</strong>, amount
```
Comment (msfeldstein) [inline src/templates/transaction/more_info.html]: Does this break the deposit flow if it was using more_info.html as well?

File: src/withdraw/tasks.py [hunk 2]
```diff
@@ -0,0 +1,68 @@
...
+    # If the Stellar transaction succeeded, we mark the corresponding `Transaction`
+    # accordingly in the database. Else, we mark it `pending_stellar`, so the wallet
+    # knows to resubmit.
+    if successful:
+        transaction.completed_at = now()
+        transaction.status = Transaction.STATUS.completed
+    else:
+        transaction.status = Transaction.STATUS.pending_stellar
+    transaction.save()
+    return True
```
Comment (msfeldstein) [inline src/withdraw/tasks.py]: you need to check that the amount and asset type is correct

File: src/templates/transaction/more_info.html (hunk 1 above)
Comment (debnil) [reply src/templates/transaction/more_info.html]: Nope - all of this information is populated in `/deposit` as well, so the resulting page looks good.

**Comment by debnil** [comment]
Merging because only a comment was added after successful runs; the build failed due to a CI configuration issue, rather than a test failure.

**Pull request merged**